### DISCUSSION TITLE
Is it possible to have multiple dynamic routes at the same level?

### DISCUSSION THREAD

**Discussion by JamesSingleton** [vercel/next.js#37968] [Help]
I have a website where I currently only have a couple of categories that look like `/category1`, `/category2`. Currently I am manually creating these pages under the `pages/` directory. I then have a `pages/[slug].tsx` which is for all the posts. Now these posts can potentially belong to multiple categories and that is why the slug does not sit under a category. My question is can I have a `pages/[category].tsx` and a `pages/[slug].tsx` or something similar to that? A follow up question is how would pagination work? I am starting to implement pagination for my category pages and created a `pages/category1/index.tsx` and a `pages/category/pages/[page].tsx` in order to facilitate pagination. Any help on structuring this would be greatly appreciated.

**Comment by ashishaugust22** [comment]
please read more about nested dynamic routing on below url
https://nextjs.org/learn/basics/dynamic-routes/dynamic-routes-details

**Reply by JamesSingleton** [reply]
There is nothing in there about have `pages/[category]/index.ts`, it talks about have a catchall route which isn't what I want to do.

**Reply by PrimozRome** [reply]
@JamesSingleton did you find any solution to this? I am stuck with the same problem...

**Reply by JamesSingleton** [reply]
@PrimozRome nope, at this point I don't think it's possible...which kind of sucks
